src/OrchardCore.Modules/OrchardCore.Users/Views/Items/CustomUserSettingsDeploymentStep.Fields.Edit.cshtml

```diff
@@ -0,0 +1,65 @@
+@model CustomUserSettingsDeploymentStepViewModel
+
+@{
+    var settingsTypeNames = Model.SettingsTypeNames;
+    var allSettingsTypeNames = Model.AllSettingsTypeNames;
+}
+
+<h5>@T["Custom User Settings"]</h5>
+
+<div class="mb-3 mt-4 mb-5">
+    <div class="row">
+        <div class="col">
+            <div class="form-check">
+                <label class="form-check-label">
+                    <input class="form-check-input" asp-for="IncludeAll"
```

Comment:
You may be able to add this to the checkbox `role="button" data-bs-toggle="collapse" data-bs-target=".cssel" aria-expanded="false" aria-controls="collapseWidthExample"`